roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -74,12 +74,20 @@
     mount_spec: true
     cifmw_cephadm_spec: "{{ cifmw_ceph_rgw_spec_path }}"
 
+- name: Generate cifmw_cephadm_assimilate_conf for ceph rgw
+  become: true
+  ansible.builtin.template:
+    src: "ceph_rgw.conf.j2"
+    dest: "{{ cifmw_cephadm_assimilate_conf }}"
```

Comment:
@fultonj this will override the generic assimilate conf (intial_ceph.conf) set at [1], is it ok ? do you see a need to use path specific to rgw like `cifmw_cephadm_rgw_assmilate.conf` and `cifmw_cephadm_rgw_assmilate.conf_container`?

[1] https://github.com/openstack-k8s-operators/ci-framework/pull/2236/files#diff-ffe549a4bf8e540dc9d3a37b219441bfaaf6d4e236bb202b6a32d55949e42f56R138

Reply:
@katarimanojk Yes, it's OK.

Note, we do not have a generic assimilate conf as you're implying. Because...

When `playbooks/ceph.yml` runs, it always generates and sets `cifmw_cephadm_bootstrap_conf` with this role.

  https://github.com/openstack-k8s-operators/ci-framework/tree/main/roles/cifmw_ceph_spec

TASK "Push it to cifmw_cephadm_assimilate_conf on bootstrap node" in pre.yml pushes it to the node and it's bootstraped. 

TASK "Generate cifmw_cephadm_assimilate_conf on bootstrap node" in pre.yml is never run because  `cifmw_cephadm_bootstrap_conf`  is never empty.

The "general" way to override the ceph.conf  in ci-framework is to use roles/cifmw_ceph_spec

Reply:
Got it, thanks.